warcio/cli.py

```diff
@@ -101,8 +101,14 @@ def get_version():
 # ============================================================================
 def indexer(cmd):
     inputs = cmd.inputs or ('-',)  # default to stdin
-    indexer = Indexer(cmd.fields, inputs, cmd.output)
-    indexer.process_all()
+    _indexer = Indexer(cmd.fields, inputs, cmd.output)
+    _indexer.process_all()
+
+
+# ============================================================================
+def checker(cmd):
+    _checker = Checker(cmd)
+    sys.exit(_checker.process_all())
```

Comment:
Is this checker code an escapee from my not-ready-to-merge warcio check branch?

Reply:
I hope not! I just changed branch to `develop`, this has been in develop since:
https://github.com/webrecorder/warcio/commit/704297b9a3d1c813dfece308314b6417bf46748d

This was the digest checker, not the more general checker that you're working on now perhaps?

Reply:
🤔 I don't see that in the diff

Reply:
Right, it shouldn't be. I switched base branch master->develop and it was probably part of the merge conflict, which is now resolved.

Reply:
I do men "check" -- the thing that's in a pull request and not ready to merge is "test"

Reply:
Ah ok! I think confusion was just that it was a merge conflict, now resolved.